workspaces/mcp-integrations/app-config.yaml

```diff
@@ -34,6 +34,7 @@ backend:
     client: better-sqlite3
     connection: ':memory:'
   auth:
+    dangerouslyDisableDefaultAuthPolicy: true
```

Comment:
hopefully they finish curating their dynamic client registration PR and we can start experimenting with enabling auth

Reply:
Updated